tensorrt_llm/scaffolding/contrib/a2a/a2a_worker.py

```
@@ -0,0 +1,89 @@
+        """
+        try:
+            loop = asyncio.get_event_loop()
+        except RuntimeError:
+            loop = None
+
+        if loop is not None and loop.is_running():
+            loop.create_task(self.async_shutdown())
+        else:
+            asyncio.run(self.async_shutdown())
+
+    task_handlers = {A2AListTask: list_handler, A2ASendTask: send_handler}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)